fe_utils/finite_elements.py

```diff
@@ -16,11 +19,106 @@ def lagrange_points(cell, degree):
     <ex-lagrange-points>`.
 
     """
-
-    raise NotImplementedError
-
-
-def vandermonde_matrix(cell, degree, points, grad=False):
+    if cell.dim > 2:
+        raise NotImplementedError(f"{dimension}D is hard :(")
+    
+    dimension = cell.dim
+    verticies = cell.vertices
+    lpoints = copy.deepcopy(verticies)
+    intervals = cell.topology[1]
```

Comment:
Don't introduce a bunch of single use variables. This does not make the code easier to read. Just use the RHS expressions.